src/components/ProductArticle.vue

```diff
@@ -6,12 +6,26 @@
         <span class="rdl-credits">Foto: Jens Jensen</span>
       </div>
     </figure>
-    <p>
-      Dato: {{ imageResult.response.docs[0].datetime }}
-    </p>
-    <p>
-      keywords: {{ imageResult.response.docs[0].keyword[0] }}
-    </p>
+    <dl>
+      <dt>Dato</dt>
+      <dd>{{ imageResult.response.docs[0].datetime }}</dd>
```

Comment:
Nothing serious but just and observation - maybe we should make a shorthand for imageResult.response.docs[0]

Reply:
Fixed using computed property.

Reply:
Nice